src/app/market/buy/checkout-process/checkout-process.component.ts

```diff
@@ -306,8 +312,13 @@ export class CheckoutProcessComponent implements OnInit, OnDestroy {
       this.snackbarService.open('Order has been successfully placed');
       this.onOrderPlaced.emit(1);
     }, (error) => {
+    if (error === errorType.itemExpired) {
```

Comment:
Since we can streamline this:

```
    if (error === errorType.itemExpired) {
      this.resetStepper();
    }
    this.snackbarService.open(error, 'warn');
    this.log.d(`Error while placing an order`);

```

The above works a little better than the if { ... } else { ... } construct